_posts/2016/2016-07-26-xconf-2016.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "XConf 2016 Hamburg"
+date: "2016-07-26 00:00:00"
+image: blog-header/lego.jpeg
```

Comment:
Please use this version for the blog-header. I resized it respectively.
![lego](https://cloud.githubusercontent.com/assets/10460958/17093369/29ef54a0-5249-11e6-9a36-3a1eb9a1bf01.jpg)